### PR TITLE
docs: fix some urls

### PR DESCRIPTION
**Pull request by eliobischof**
- token endpoint is /oauth/v2/oauth/token
- zitadel.dev is internal only

### CODE REVIEW

File: docs/docs/quickstarts/login/nextjs.md [hunk 1]
```diff
@@ -50,10 +50,10 @@ export const ZITADEL = {
...
     params: { response_type: "code", grant_type: "authorization_code" },
     authorizationParams: { grant_type: "authorization_code", response_type: "code" },
-    accessTokenUrl: "https://api.zitadel.dev/oauth/v2/token",
-    requestTokenUrl: "https://api.zitadel.dev/oauth/v2/token",
-    authorizationUrl: "https://accounts.zitadel.dev/oauth/v2/authorize",
-    profileUrl: "https://api.zitadel.dev/oauth/v2/userinfo",
+    accessTokenUrl: "https://api.zitadel.ch/oauth/v2/oauth/token",
+    requestTokenUrl: "https://api.zitadel.ch/oauth/v2/oauth/token",
+    authorizationUrl: "https://accounts.zitadel.ch/oauth/v2/authorize",
+    profileUrl: "https://api.zitadel.ch/oauth/v2/userinfo",
```
Comment (hifabienne) [inline docs/docs/quickstarts/login/nextjs.md:56]: this urls are all wrong in my opinion. should be with {your_domain}

**Comment by mffap** [comment]
@eliobischof could you have a look at the deploy logs? There are many broken links flagged. This PR seems related.

**Review by livio-a — CHANGES_REQUESTED** [review]
token endpoint has been fixed and is now /oauth/v2/token again

**Comment by mffap** [comment]
@livio-a @eliobischof do we have an issue right now with different endpoints? or is the current version of the docs (without) this PR correct? This PR is confusing in that regard.

**Comment by eliobischof** [comment]
Token endpoint changes are obsolete and for ZITADEL domains, we need a better solution than this PRs.
Closing.

**Pull request closed without merging**